test/xray/generator.go

```diff
@@ -55,6 +56,7 @@ func newLoadGenerator(cfg *common.TraceGeneratorConfig) *XrayTracesGenerator {
 }
 func (g *XrayTracesGenerator) Generate(ctx context.Context) error {
 	rootCtx, root := xray.BeginSegment(ctx, "load-generator")
+	log.Println("\033[34mGenerated Trace\033[0m")
```

Comment:
This seems like it's going to pollute the logs especially for high loads (e.g. 5000).